lib/src/_shared/PickerPopper.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import Grow from '@material-ui/core/Grow';
+import Paper, { PaperProps } from '@material-ui/core/Paper';
+import Popper, { PopperProps } from '@material-ui/core/Popper';
+import TrapFocus, { TrapFocusProps } from '@material-ui/core/Unstable_TrapFocus';
+import { useForkRef } from '@material-ui/core/utils'
+import { makeStyles } from '@material-ui/core/styles';
+import { useGlobalKeyDown, keycode } from './hooks/useKeyDown';
+import { IS_TOUCH_DEVICE_MEDIA } from '../constants/dimensions';
+import { executeInTheNextEventLoopTick } from '../_helpers/utils';
+import { TransitionProps } from '@material-ui/core/transitions/transition';
+
+export interface ExportedPickerPopperProps {
+  /**
+   * Popper props passed down to material-ui [Popper](https://material-ui.com/api/popper/#popper-api) component.
+   */
+  PopperProps?: Partial<PopperProps>;
+  /**
+   * Custom component for [Transition](https://material-ui.com/components/transitions/#transitioncomponent-prop).
+   */
+  TransitionComponent?: React.ComponentType<TransitionProps>;
+}
+
+export interface PickerPopperProps extends ExportedPickerPopperProps, PaperProps {
+  role: 'tooltip' | 'dialog';
+  TrapFocusProps?: Partial<TrapFocusProps>;
+  anchorEl: PopperProps['anchorEl'];
+  open: PopperProps['open'];
+  onClose: () => void;
+}
+
+export const useStyles = makeStyles(
+  theme => ({
+    popper: {
+      zIndex: theme.zIndex.modal,
+    },
+    paper: {
+      transformOrigin: 'top center',
+      '&:focus': {
+        outline: 'auto',
+        [IS_TOUCH_DEVICE_MEDIA]: {
+          outline: 0,
+        },
+      },
```

Comment:
We can remove redundant default for simplicity until we solve the root issue.

```suggestion
      [IS_TOUCH_DEVICE_MEDIA]: {
        outline: 0,
      },
```

Reply:
Bump

Reply:
bump